.circleci/config.yml

```diff
@@ -1,15 +1,18 @@
 version: 2.1
 orbs:
   ruby: circleci/ruby@1.1.1
+  buildevents: honeycombio/buildevents@0.5
+  carwow: carwow/carwow-orb@0.4
 
 jobs:
   build:
-    docker:
-      - image: circleci/ruby:2.6.3-stretch-node
-
+    executor: carwow/ruby
     steps:
       - checkout
+      - run: sudo apt-get update
       - run: sudo apt-get install unixodbc-dev
+      - run: sudo apt-get autoclean
+      - run: sudo apt-get clean
```

Comment:
What's the aim here?

My concern is that it's just using up extra time, considering we're a) charged per minute and b) are sometime sitting and waiting for a build to finish. Does cleaning up the repos do anything?